draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1429,11 +1469,20 @@ material {{MOV96}} {{CFADLNV05}}.
 
 -   inv0(x): This function returns the multiplicative inverse of x in F,
     extended to all of F by fixing inv0(0) == 0.
-    To implement inv0 in constant time, compute inv0(x) := x^(q - 2).
-    Notice on input 0, the output is 0 as required.
+    A straightforward way to implement inv0 in constant time is to compute
+
+    ~~~
+    inv0(x) := x^(q - 2).
+    ~~~
+
+    Notice that on input 0, the output is 0 as required.
+    Certain fields may allow faster inversion methods; detailed discussion
+    of such methods is beyond the scope of this document.
 
 -   I2OSP and OS2IP: These functions are used to convert a byte string to
     and from a non-negative integer as described in {{!RFC8017}}.
+    (Note that these functions operate on byte strings in big endian byte
```

Comment:
Maybe network byte order? (Unclear if that's preferred for IETF documents.)

Reply:
RFC7748 uses "little endian", as does the current Ristretto draft. Other RFCs are mixed, but use of "little-endian" and "big-endian" seems to be pretty common even in older RFCs (e.g., RFC1071).

My guess is that implementors of this spec are more likely to understand "big endian" than "network byte order," so I'd vote to keep as-is. Thoughts?

Reply:
I strongly oppose any use similar to `htonl` / `ntohl` (host to network long and network to host long), those are machine dependent / implementation-defined and lead to situation like the C `int` datatypes which is at least 16-bit but almost always assume to be 32-bit i.e. `int16_t` and `int32_t` is clearer and less error prone (and for the spec big endian).

Reply:
Great point! Let's keep it as is.